drake/common/eigen_types.h

```diff
@@ -50,6 +50,10 @@ using Matrix3X = Eigen::Matrix<Scalar, 3, Eigen::Dynamic>;
 template <typename Scalar>
 using Matrix4X = Eigen::Matrix<Scalar, 4, Eigen::Dynamic>;
 
+/// A matrix of 6 rows, dynamic columns, templated on scalar type.
+template <typename Scalar>
+using Matrix6X = Eigen::Matrix<Scalar, 6, Eigen::Dynamic>;
```

Comment:
Added this because I thought that referring to a matrix containing wrenches as a `TwistMatrix<Scalar>` was weird.